package app.exception;

class ExceptionApplicationTests {


	void contextLoads() {
	}

}
